{
	"id": "oval:org.opensuse.security:def:20142667",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2014-2667",
		"affected": {
			"family": "unix",
			"platform": [
				"openSUSE 13.2"
			]
		},
		"reference": [
			{
				"ref_id": "CVE-2014-2667",
				"ref_url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-2667",
				"source": "CVE"
			}
		],
		"description": "\n    Race condition in the _get_masked_mode function in Lib/os.py in Python 3.2 through 3.5, when exist_ok is set to true and multiple threads are used, might allow local users to bypass intended file permissions by leveraging a separate application vulnerability before the umask has been set to the expected value.\n    ",
		"advisory": {
			"issued": {},
			"updated": {},
			"affected_cpe_list": {}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:org.opensuse.security:tst:2009137614",
						"comment": "libpython3_4m1_0-3.4.1-4.2.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009137615",
						"comment": "python3-base-3.4.1-4.2.1 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009117729",
				"comment": "openSUSE 13.2 is installed"
			}
		]
	}
}
